psi4/driver/procrouting/proc.py

```diff
@@ -3061,6 +3098,269 @@ def run_adc(name, **kwargs):
     return core.adc(ref_wfn)
 
 
+def run_adcc(name, **kwargs):
+    """Prepare and run an ADC calculation in adcc, interpret the result and return
+    as a wavefunction.
+
+    """
+    # TODO Maybe it would improve readability if this function was spilt
+    #      up and the whole thing went to a separate file (like for sapt,
+    #      interface_cfour.py, ...
+
+    if not extras.addons("adcc"):
+        # TODO or PsiImportError ???
+        raise ValidationError("adcc extras module not available")
```

Comment:
""adcc extras module not available, please install via ..."

```
try:
    import adcc
except ModuleNotFoundError:
    raise ...

Reply:
Good point. Thanks.

Reply:
Fixed now.